.github/workflows/e2e.yaml

```diff
@@ -30,17 +32,17 @@ jobs:
           sudo mv linux-amd64/helm /usr/local/bin/helm
           sudo mv linux-amd64/tiller /usr/local/bin/tiller
       - name: Setup Minikube
-        uses: manusa/actions-setup-minikube@v2.3.1
+        uses: manusa/actions-setup-minikube@v2.4.0
         with:
-          minikube version: v1.19.0
+          minikube version: v1.20.0
           kubernetes version: ${{ matrix.k8sVersion }}
           driver: none # required for nri-kubernetes, as metrics gathered for the host are tested as well
       - name: Compile and Build Image
         run: |
           GOOS=linux GOARCH=amd64 make compile # Set GOOS and GOARCH explicitly since Dockerfile expects them in the binary name
           docker build -t test_image_${{ matrix.privilegedMode }}:test --build-arg 'MODE=${{ matrix.privilegedMode }}' .
       - name: Interact with the cluster
-        continue-on-error: ${{ contains(matrix.k8sVersion, '1.20') }} # experimental support for 1.20.x
+        continue-on-error: ${{ contains(matrix.k8sVersion, 'v1.22') }} # Mark best-effort (but unsupported) minors here
```

Comment:
Currently it does not have any effect since that version is not in the matrix, I am fine with either options.
I would add 1.22 but it still not available, so I would remove or comment the line

Reply:
Done!